ozone_concentrations/create_artifact.jl

```diff
@@ -0,0 +1,81 @@
+using Dates
+using Downloads
+using NCDatasets
+
+using ClimaArtifactsHelper
+
+FILE_URLs = [
+    "https://caltech.box.com/shared/static/hj2yucye0u3l8e4x8o1d0w35ijj5amh6.nc",
+    "https://caltech.box.com/shared/static/h3c8rlpctxpq3uh1r01zhips426lgi0e.nc",
+            ]
+FILE_PATHs = [
+    "vmro3_input4MIPs_ozone_CMIP_UReading-CCMI-1-0_gn_195001-199912.nc",
+    "vmro3_input4MIPs_ozone_CMIP_UReading-CCMI-1-0_gn_200001-201412.nc",
+             ]
+
+const H_EARTH = 7000.0
+const P0 = 1e5
+const HPA_TO_PA = 100.0
+
+Plvl_inv(P) = -H_EARTH * log(P / P0)
+
+function create_ozone(infile_paths, outfile_path)
+    FT = Float64
+
+    ncin = NCDataset(infile_paths, aggdim = "time")
+    ncout = NCDataset(outfile_path, "c")
+
+    defDim(ncout, "lon", length(ncin["lon"]))
+    defDim(ncout, "lat", length(ncin["lat"]))
+    defDim(ncout, "z", length(ncin["plev"]))
+    defDim(ncout, "time", length(ncin["time"]))
+
+    lon = defVar(ncout, "lon", FT, ("lon",), attrib = ncin["lon"].attrib)
+    lon[:] = Array(ncin["lon"])
+
+    lat = defVar(ncout, "lat", FT, ("lat",), attrib = ncin["lat"].attrib)
+    lat[:] = Array(ncin["lat"])
+
+    z = defVar(ncout, "z", FT, ("z",))
+    plevin = ncin["plev"][:] .* HPA_TO_PA
+    @. z = Plvl_inv(plevin)
+    z.attrib["long_name"] = "altitude"
+    z.attrib["units"] = "meters"
+
+    time_ = defVar(ncout, "time", FT, ("time",), attrib = ncin["time"].attrib)
+    time_[:] = Array(ncin["time"])
+
+    defVar(
+        ncout,
+        "vmro3",
+        FT,
+        ("lon", "lat", "z", "time"),
+        attrib = filter(((k, v),) -> k != "_FillValue", ncin["vmro3"].attrib),
```

Comment:
Maybe add a comment on why we filter out `_FillValue`?

Reply:
I tried it again without the filter and it works now, so I removed it. (Not sure what happened before, maybe I was using a different version of some packages).